src/reflect/scala/reflect/internal/Symbols.scala

```diff
@@ -725,7 +725,7 @@ trait Symbols extends api.Symbols { self: SymbolTable =>
 
     def setFlag(mask: Long): this.type   = { _rawflags |= mask ; this }
     def resetFlag(mask: Long): this.type = { _rawflags &= ~mask ; this }
-    def resetFlags() { rawflags &= TopLevelCreationFlags }
+    def resetFlags() { rawflags = 0 }
```

Comment:
right. cc @adriaanm - what do you think about this change? note the commit message, which says:

```
clear all flags when resetting a symbol
this change is a bit scary because it changes code that's not been
changed in 11 years
7fa7c93#diff-d5789e5ae5061197d782d08324b260dbL214
```